src/shared/utilities/cliUtils.ts

```diff
@@ -106,9 +102,7 @@ export async function installCli(
             if (selection === manualInstall) {
                 vscode.env.openExternal(vscode.Uri.parse(cliToInstall.manualInstallLink))
             }
-            result = 'Cancelled'
-
-            return undefined
+            throw new TimeoutError('cancelled')
```

Comment:
This shouldn't be considered a timeout.

Reply:
Yes I know it should be renamed. The class itself should've never been named `Timeout`; it was really a cancellation token this whole time.